### PR TITLE
PHP7 raffler

### PR DESCRIPTION
**Pull request by WyriHaximus**
Because the default PHP5 raffler is rather boring.

Progress:
- [x] docker image
- [x] PHP7 Raffler

### CODE REVIEW

**Comment by pyrech** [comment]
Please add the return type to be truly php 7 ! :)

**Comment by WyriHaximus** [comment]
Added :+1: ! Doesn't work on 3v4l.org yet but going to get a install.sh done soon :D.

**Comment by pyrech** [comment]
\o/

**Comment by dennisdegreef** [comment]
Works now :)

https://3v4l.org/O7IU5

**Comment by lucasvanlierop** [comment]
@WyriHaximus Think it's about time to get this thing merged :-)

Would you mind add some docker config for it and adding it to:
* https://github.com/domcode/rafflers/blob/master/build.sh
* https://github.com/domcode/rafflers/blob/master/raffle.sh

Check one of the existing rafflers for inspiration, or ping me directly.

**Comment by WyriHaximus** [comment]
@lucasvanlierop Updated :D!

File: wyrihaximus-php7/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+FROM php:7-cli
+MAINTAINER lucas@vanlierop.org
```
Comment (lucasvanlierop) [inline wyrihaximus-php7/Dockerfile]: Give yourself some credit yo! :-)

**Comment by WyriHaximus** [comment]
Pulled from upstream and merged the latest changed

**Pull request merged**